runtime/crab/src/lib.rs

```diff
@@ -93,8 +93,14 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	Migrations,
 >;
 
+type Migrations = (
+	// pallet_balances::migration::MigrateToTrackInactive<Runtime, CheckingAccount>,
```

Comment:
From https://github.com/paritytech/substrate/pull/12813
> For chains with XCM, this should be used to mark the Checking Account when upgraded.

@jiguantong @AurevoirXavier Please help me. I am not sure if we should apply the `MigrateToTrackInactive` migration in this case.